src/routes/components/components.json

```diff
@@ -1002,13 +1003,6 @@
 		"title": "layercake",
 		"repository": "https://github.com/mhkeller/layercake"
 	},
-	{
-		"category": "CSS and Layout",
-		"description": "resizable layout component for svelte",
-		"tags": ["components and libraries", "layout and structure"],
-		"title": "svelte-layout-resizable",
-		"repository": "https://github.com/milahu/svelte-layout-resizable"
-	},
```

Comment:
Why removing this package ?

Reply:
I couldn't find the package on npm - please let me know if you can find it!

Reply:
The package have been deprecated, but can still install it with:
```shell
npm install https://github.com/milahu/svelte-layout-resizable
# or
npm install milahu/svelte-layout-resizable
```
(ref: https://docs.npmjs.com/cli/v10/commands/npm-install)

But, do we want to allow that ? :thinking:

Reply:
The package itself can't be viewed if you visit https://npmjs.com/package/@milahu/svelte-layout-resizable since it's a private package (https://npmjs.com/package/milahu/svelte-layout-resizable is just a 404) - and since it's been deprecated anyway, probably not?